python_training/mimic.py

```diff
@@ -0,0 +1,122 @@
+
+class MyList:
```

Comment:
Looks really good overall!
But one important method is still missing - `my_list[0]` should return the element in this index (or an exception).